doc/orchestrator.rst

```diff
@@ -165,3 +165,460 @@ A full example of configuring KeyDB to run in SmartSim is shown below
+
+The ``Experiment.start()`` function launches the ``Orchestrators`` for use within the workflow. In other words, the function
+deploys the databases on the allocated compute resources.
+
+.. note::
+  By setting `summary=True`, we can see a summary of the
+  experiment printed before it is launched. The summary
+  will stay for 10 seconds, and it is useful as a last
+  check. If we set `summary=False`, then the experiment
+  would be launched immediately.
+
+Initialize Clients
+^^^^^^^^^^^^^^^^^^
+The SmartRedis ``Client`` object contains functions that manipulate, send, and receive
+data within the database. Each database has a single, dedicated SmartRedis ``Client``.
+Begin by initializing a SmartRedis ``Client`` object per launched database.
+
+To create a designated SmartRedis ``Client``, you need to specify the address of the target
+running database. You can easily retrieve this address using the ``Orchestrator.get_address()`` function.
+
+For the single-sharded database:
+
+.. code-block:: python
+
+  driver_client_single_shard = Client(cluster=False, address=single_shard_db.get_address()[0], logger_name="Single shard db logger")
+
+For the multi-sharded database:
+
+.. code-block:: python
+
+  driver_client_multi_shard = Client(cluster=True, address=multi_shard_db.get_address()[0], logger_name="Multi shard db logger")
+
+Send Data
+^^^^^^^^^
+In the application script, we retrieved two NumPy tensors.
+To support the apps functionality, we will create two
+NumPy arrays in the python driver script and send them to the a database. To
+accomplish this, we use the ``Client.put_tensor()`` function with the respective
+database client instances.
+
+For the single-sharded database:
+
+.. code-block:: python
+
+  array_1 = np.array([1, 2, 3, 4])
+  driver_client_single_shard.put_tensor("tensor_1", array_1)
+
+For the multi-sharded database:
+
+.. code-block:: python
+
+  array_2 = np.array([5, 6, 7, 8])
+  driver_client_multi_shard.put_tensor("tensor_2", array_2)
+
+Lets check to make sure the database tensors do not exist in the incorrect databases:
+
+.. code-block:: python
+
+  check_single_shard_db_tensor_incorrect = driver_client_single_shard.key_exists("tensor_2")
+  check_multi_shard_db_tensor_incorrect = driver_client_multi_shard.key_exists("tensor_1")
+  logger.info(f"The multi shard array key exists in the incorrect database: {check_single_shard_db_tensor_incorrect}")
+  logger.info(f"The single shard array key exists in the incorrect database: {check_multi_shard_db_tensor_incorrect}")
+
+When you run the experiment, the following output will appear:
+
+.. code-block:: bash
+
+  00:00:00 system.host.com SmartSim[#####] INFO The multi shard array key exists in the incorrect database: False
+  00:00:00 system.host.com SmartSim[#####] INFO The single shard array key exists in the incorrect database: False
+
+Initializing the colo Model
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+In the next stage of the experiment, we will utilize the
```

Comment:
Reference where the application script was written before

Reply:
Good idea!